src/wp-content/themes/twentynineteen/style-editor.scss

```
@@ -1020,3 +1020,15 @@ $group-block-background__padding: $font__size_base;
 		width: 100%;
 	}
 }
+
+/** === Post Author Block === */
+
+.editor-styles-wrapper {
+	
+	& .avatar,
+	& .wp-block-post-author__avatar img {
+		border-radius: 50%;
+	}
+
+}
```

```suggestion
.avatar,
.wp-block-post-author__avatar img {
	border-radius: 100%;
}
```

Twenty Nineteen's editor styles automatically add the `.editor-styles-wrapper` selector. Also, while 50% may look like 100%, Twenty Nineteen uses 100% for the front end.

---

@sabernhardt I have updated changes as per mentioned.